### PR TITLE
Standard components setup

### PR DESCRIPTION
**Pull request by Cleop**
- Sets up style guide page and route
- Sets up tachyons, #33 
- Creates first standard components: primary button, secondary button, colour palette #34

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,3 +23,6 @@ erl_crash.dump
 # variables.
 /config/prod.secret.exs
 cover
+
+# allow npm to get the last version of tachyons see also postinstall in package.json
+	web/static/css/tachyons.min.css
```
Comment (SimonLab) [inline .gitignore]: web/static/css/tachyons.min.css is already part of the repo, not sure if you want to keep the file?

File: web/templates/style_guide/index.html.eex [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+          <h2 class="f3">Buttons</h1>
+            <p class="measure f5 lh-copy sans-serif">Used to create a styled <code>&lt;a&gt;</code>, <code>&lt;button&gt;</code>, or <code>&lt;input&gt;</code> element.</p>
+
+            <%= App.ComponentHelpers.component "primary_button", value: "Primary" %>
+            <div class="dib w-20 pv3 ma2 lm-bg-dark-blue tc">
+              <%= App.ComponentHelpers.component "secondary_button", value: "Secondary" %>
+            </div>
+
+            <!-- <h2 class="f3">Links</h2>
```
Comment (SimonLab) [inline web/templates/style_guide/index.html.eex]: Will this commented html be use in a near future? I prefer to delete all commented code, it make the repo clean and avoid confusion.

**Review by SimonLab — APPROVED** [review]
Looks good to me. Just 2 small comments but I let you decide what you want to do with them :+1:

File: .gitignore (hunk 1 above)
Comment (Cleop) [inline .gitignore]: Cool, I will delete the file but keep this code in the .gitignore

**Review by Danwhy — APPROVED** [review]

**Pull request merged**